packages/react-signer/src/Modal.tsx

```diff
@@ -183,25 +206,15 @@ class Signer extends React.PureComponent<Props, State> {
     const { isExternal, isHardware, hardwareType } = extractExternal(currentItem.accountId);
 
     return (
-      <Modal.Actions
-        onCancel={
-          isQrVisible
-            ? this.onCancelQr
-            : this.onCancel
-        }
-      >
-        {!isRenderError && (!isQrVisible || !isQrScanning) && (
+      <Modal.Actions cancelLabel={signedTx ? t('Close') : undefined} withOr={!signedTx} onCancel={isQrVisible ? this.onCancelQr : signedTx ? this.onCancelSign : this.onCancel}>
```

Comment:
Ahhh... ignore the previous comment on `Cancel`, I see you have cleaned this on up. All ok.